src/rdb.c

```diff
@@ -3074,8 +3074,11 @@ int rdbLoadRioWithLoadingCtxScopedRdb(rio *rdb, int rdbflags, rdbSaveInfo *rsi,
     return retval;
 }
 
-/* Load an RDB file from the rio stream 'rdb'. On success C_OK is returned,
- * otherwise C_ERR is returned.
+/* Load an RDB file from the rio stream 'rdb'. We return one of the following:
+ * - RDB_OK On success
+ * - RDB_INCOMPATIBLE If the RDB is has an invalid signature or version
+ * - RDB_FAILED
+ * if the RDB is has an invalid signature or version we return 
```

Comment:
This seems duplicate of the above. We could maybe mention in all other failure cases.